### PR TITLE
Update RenderNotification particle to skip standard template&model rendering

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

**Comment by sjmiles** [comment]
Funneling the logic into one place (renderOutput or update) is best-practice but it's not obvious until one has to coordinate multiple asynchronous signals. I haven't been talking about this much before, but I plan to mention it more as we build code that others will use as examples.

**Comment by mmandlis** [comment]
> Off the top of my head:
> 
> ```
>   const notification = {
>     modality: 'notification',
>     title: `Hello`,
>     text: `I'm a notification.`,
>     handler: `onNotificationClick`
>   };
>   
>   const clickNotification = {
>     modality: 'notification',
>     title: `Hello`,
>     text: `Notification was clicked`
>   };
>   
>   return class extends UiParticle {
>     renderOutput(inputs, state) {
>       if (!state.notified) {
>         state.notified = true;
>         this.output(notification);
>       }
>       if (state.clicked) {
>         state.clicked = true;
>         this.output(clickNotification);
>       }
>     }
>     onNotificationClick(event) {
>       this.state = {clicked: true};
>     }
>   };
> ```

Thanks, this is so much better, I updated the particle, PTAL

**Comment by sjmiles** [comment]
Glad that worked for you.

File: particles/PipeApps/source/RenderNotification.js [hunk 1]
```diff
@@ -12,21 +12,32 @@
...
-        text: `I'm a notification.`,
-        handler: `onNotificationClick`
-      });
+    renderOutput(inputs, state) {
+      if (!state.notified) {
+        state.notified = true;
+        this.output(notification);
+      }
+      if (state.clicked) {
+        state.clicked = true;
```
Comment (sjmiles) [inline particles/PipeApps/source/RenderNotification.js]: Should be `state.clicked = false`

Comment (mmandlis) [reply particles/PipeApps/source/RenderNotification.js]: oops, this is what happens when i copy&paste without paying attention :) thanks for noticing, fixed!

**Pull request merged**